tests/integration/runners/test_fileserver.py

```diff
@@ -161,6 +163,10 @@ def test_file_list(self):
         self.assertIsInstance(ret['return'], list)
         self.assertTrue('grail/scene33' in ret['return'])
 
+    # Git doesn't handle symlinks in Windows. See the thread below:
+    # http://stackoverflow.com/questions/5917249/git-symlinks-in-windows
+    @skipIf(salt.utils.is_windows(),
+            'Git doesn\'t handle symlinks properly on Windows')
```

Comment:
It's when you clone the salt repository initially. There is a symink file in `......\salt\tests\integration\files\file\base`. It's named `dest_sym` and I think it's supposed to be linked to `source_sym`. When you clone in Windows, these don't get created as symlinks, but rather as files. So, the test suite is expecting to find a symlink in this directory, but it doesn't. I tried the newest git and it did create `dest_sym` as a symlink, but it was empty... so there are still some problems with it.